schema/query.go

```diff
@@ -354,25 +358,33 @@ func (e NotExist) Match(payload map[string]interface{}) bool {
 // Match implements Expression interface
 func (e GreaterThan) Match(payload map[string]interface{}) bool {
 	n, ok := isNumber(getField(payload, e.Field))
-	return ok && (n > e.Value)
+	if ok {
+		return ok && (n > e.Value.(float64))
+	}
+	t, ok := isTime(getField(payload, e.Field))
+	if ok {
+		t2, ok := isTime(e.Value)
+		return ok && (t.After(t2))
+	}
+	return false
 }
 
 // Match implements Expression interface
 func (e GreaterOrEqual) Match(payload map[string]interface{}) bool {
```

Comment:
Whoops.  Should have pushed those as well.  I'll correct this once I finalize the other fixes.